controls/scripts/ControlsPubNode.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+import rospy
+from std_msgs.msg import Header
+from ControlsPublisher import ControlsPublisher
+from geometry_msgs.msg import WrenchStamped
+from geometry_msgs.msg import Wrench
+from geometry_msgs.msg import Vector3
+from controls.msg import ThrusterForceArray
+from controls.msg import ThrusterPWMArray
+
+def controlsPubNode():
+    rospy.init_node('ControlsPubNode', anonymous=True)
```

Comment:
ros convention says to name nodes lower case and snake case. I know this isn't consistent with our own style, but that's what they want and it'll give weird warnings when you don't do it. So this should just be `controls_pub_node`.